### PR TITLE
Use non-depricated events for loading additional scripts in files app

### PR DESCRIPTION
**Pull request by icewind1991**
Removes some log spam

### CODE REVIEW

File: apps/files_external/lib/Service/BackendService.php [hunk 1]
```diff
@@ -109,9 +110,8 @@ public function registerBackendProvider(IBackendProvider $provider) {
 	private function callForRegistrations() {
 		static $eventSent = false;
 		if(!$eventSent) {
-			\OC::$server->getEventDispatcher()->dispatch(
-				'OCA\\Files_External::loadAdditionalBackends'
-			);
+			$event = new LoadAdditionalScriptsEvent();
+			\OC::$server->getEventDispatcher()->dispatch(LoadAdditionalScriptsEvent::class, $event);
```
Comment (ChristophWurst) [inline apps/files_external/lib/Service/BackendService.php]: is this the same as `loadAdditionalBackends`?

Comment (icewind1991) [reply apps/files_external/lib/Service/BackendService.php]: fixed :see_no_evil:

**Review by ChristophWurst — APPROVED** [review]
Looks good!

**Review by rullzer — APPROVED** [review]
sure

**Pull request merged**